content/docs/getting_started/visual_story.md

```diff
@@ -19,8 +19,8 @@ In this tutorial, we'll introduce you to the [amp-story](/docs/reference/compone
 - Animate elements on a page
 - Keep readers engaged with your content by adding related links to the end of the story
 
-Note: The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**.
+{% call callout('Note', type='note') %} The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**. {% endcall %}
```

Comment:
Why did you revert to this syntax? Was this an oversight? The callouts are not required anymore for simple notes.